pkg/controller/jobframework/integrationmanager.go

```diff
@@ -147,7 +149,15 @@ func (m *integrationManager) getExternal(kindArg string) (runtime.Object, bool)
 	return jt, f
 }
 
+func (m *integrationManager) getEnabledIntegrations() set.Set[string] {
```

Comment:
```suggestion
func (m *integrationManager) enabledIntegrations() set.Set[string] {
```

Reply:
This might not be possible, as that already exists [here](https://github.com/kubernetes-sigs/kueue/pull/2574/files#diff-fde4763cb4c71b0994c0012fc717450c2c2d90a182e3fa30f3d5a21417b42686L150-L152). WDYT?

Reply:
The name `enableIntegration` is slightly different so will not conflict with `enabledIntegrations`.

Reply:
Sorry, I meant `enabledIntegrations` would conflict with this[ `enabledIntegrations`](https://github.com/kubernetes-sigs/kueue/pull/2574/files#diff-fde4763cb4c71b0994c0012fc717450c2c2d90a182e3fa30f3d5a21417b42686R85)